docs/nodejs-compatibility.md

```diff
@@ -12,6 +12,10 @@ The instrumentation is automatic.
 | -------------------------- | :-----------------: | :----: | :-----: | :--: |
 | [Jest](https://jestjs.io/) |          ✓          |   ✓    |    ✓    |  ✓   |
 
+### HTTP libraries
+
+All http libraries that rely on [Node.js http](https://nodejs.org/api/http.html) are automatically instrumented.
```

Comment:
This should be a table, like https://docs.scope.dev/docs/go-compatibility#libraries
Native libraries show like third party libraries.

Reply:
Also, is that the client, or the server, or both?

Reply:
both

Reply:
well, actually both in testing. In runtime only server.